kafka-ui-contract/src/main/resources/swagger/kafka-connect-api.yaml

```diff
@@ -0,0 +1,495 @@
+openapi: 3.0.0
+info:
+  description: Api Documentation
+  version: 0.1.0
+  title: Api Documentation
+  termsOfService: urn:tos
+  contact: {}
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+tags:
+  - name: /connect
+servers:
+  - url: /localhost
+
+paths:
+  /connectors:
+    get:
+      tags:
+        - /connect
```

Comment:
tags looks strange, why is it like a path?

Reply:
What do you suggest for this one?

Reply:
It shouldn't have a slash. That's it :)